### ISSUE TITLE
Feature request: dont' close the plugin window

### ISSUE THREAD

**Issue by panwooders** [keremciu/sketch-iconfont#89]
Every time you add an icon, the plugin windows closes and you must open the topmenu.
It could be great if we could add multiple icons without closing the window.
Thanks! I love your plugin

**Comment by keremciu** [comment]
@panwooders yep, I added this functionality to the new version you can follow it on #85 I hope it will be released so soon

**Comment by saxmode** [comment]
Hey

Your plugin is awesome. But I've got exactly the same problem as panwooders. If I want to insert about 6-10 icons the plugin is just not practical. I've got to click 2 times and make 4 decisions like: «Plugins / IconFonts / GridInsert / IconsetXY» to insert one icon. If I do this for 10 Icons.. it's easier to copy the icons from the fontawesome cheatsheet (which has no search functionality but the browsers one). I propose you should just don't close the window on click on icon. Just leave it there and give me a button to close the window myself. 

Also it would be awesome if you could resize the window.

If this is fixed, I'd buy you more than one beer - let me tell you.

Best regards
Nicolas

**Comment by keremciu** [comment]
Hey @saxmode 

yea, I could add a new window mode which is has a close [x] button on window and you can select multiple icons, I will try to add that one this weekend :) thank you a lot ✌🏻

Cheers,
Kerem

**Comment by saxmode** [comment]
Thank YOU ;).

Cheers - enjoy the pint.

Best
Nicolas

**Comment by gabrielvaz** [comment]
The plugin is awesome, but this issue will do a lot of difference on my workflow!
